libPhoneNumber/NBPhoneNumberOfflineGeocoder.h

```diff
@@ -0,0 +1,40 @@
+//
+//  NBPhoneNumberOfflineGeocoder.h
+//  libPhoneNumberiOS
+//
+//  Created by Rastaar Haghi on 6/12/20.
+//  Copyright © 2020 ohtalk.me. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "NBGeocoderMetadataHelper.h"
```

Comment:
Add an empty new line after the framework import before local header imports

Reply:
done.